src/treemodel/position.js

```diff
@@ -406,7 +448,17 @@ CKEDITOR.define( [ 'treemodel/rootelement', 'utils', 'ckeditorerror' ], ( RootEl
 
 			path.push( offset );
 
-			return new Position( parent.root, path );
+			return new this( parent.root, path );
+		}
+
+		/**
+		 * Creates and returns a new instance of Position, which is equal to passed position.
+		 *
+		 * @param {treeModel.Position} position Position to be cloned.
+		 * @returns {treeModel.Position}
+		 */
+		static createFromPosition( position ) {
```

Comment:
Ah... Then I don't have a strong opinion :) Keeping `createFromXXX` will be consistent with other methods which we already have and changing to `clone` will make it look better. Let's keep what we have now.